
[Executed at: Mon Apr 9 5:51:23 PDT 2018]

1.414
Traceback (most recent call last):
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24781_3/asnlib/autograde.py", line 14, in <module>
    score=runtest.main()
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24781_3/asnlib/runtestcases.py", line 77, in main
    stu_work=getattr(__import__(work_package, fromlist=fun_name), f)
  File "./q5_work.py", line 10, in <module>
    print(nroot_complex(2,5,-4))
  File "./q5_work.py", line 8, in nroot_complex
    y = x*1j
NameError: name 'x' is not defined
Command exited with non-zero status 1
